extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -953,10 +955,12 @@ export class AmpA4A extends AMP.BaseElement {
    * @return {!Promise} awaiting load event for ad frame
    * @private
    */
-  iframeRenderHelper_(iframe) {
+  iframeRenderHelper_(iframe, sentinel) {
```

Comment:
nit: for better readability can we move this method after `renderViaNameAttrOfXOriginIframe_`.